app/styles/_shame.scss

```diff
@@ -50,4 +50,8 @@ code {
 
 .au-o-grid {
   display: block;
-}
\ No newline at end of file
+}
+
+.playground__edit {
```

Comment:
I guess this belongs in the `c-playground.scss` file?

Reply:
The buttons disappeared when adding more fields but added a margin now for the `au-o-box` in the playground that fixes it for the builder form and the validation form 
```css
.au-o-box {
  margin-bottom: 4rem;
}

Reply:
Hmmmmmmmmm 😄, I wouldn't extend Appuniversum classes like that since it will also affect other cases (or future cases we might add). We probably can fix it by restructuring the html setup, but as a quick fix I would use one of the margin spacing utils: https://appuniversum.github.io/ember-appuniversum/?path=/story/utilities-spacing--page

Reply:
Woeps haha

It is fixed with the spacing style on the builder form but not yet fully happy about it in the validation form. But it "workable"